packages/cli/src/commands/database/seed/tables.ts

```diff
@@ -114,14 +115,18 @@ export const seedTables = async (
   latestTimestamp: number
 ) => {
   await createTenant(connection, defaultTenantId);
+  await seedOidcConfigs(connection, defaultTenantId);
+  await seedAdminData(connection, defaultManagementApi);
+
+  await createTenant(connection, adminTenantId);
+  await seedOidcConfigs(connection, adminTenantId);
+  await seedAdminData(connection, createManagementApiInAdminTenant(defaultTenantId));
+  await seedAdminData(connection, createMeApiInAdminTenant());
+
   await Promise.all([
-    connection.query(insertInto(managementResource, 'resources')),
-    connection.query(insertInto(managementResourceScope, 'scopes')),
     connection.query(insertInto(createDefaultAdminConsoleConfig(), 'logto_configs')),
     connection.query(insertInto(defaultSignInExperience, 'sign_in_experiences')),
     connection.query(insertInto(createDemoAppApplication(generateStandardId()), 'applications')),
```

Comment:
Why do we not create these data for the admin tenant?

Reply:
@xiaoyijun check seedAdminData details below

Reply:
Sorry about the confusion.
I mean `sign-in-experiences` and `applications`.

Reply:
yes i'll do that